docs/book/src/development/ami-inventory.md

```
@@ -26,35 +26,1095 @@ CAPA AMIs should be published with these configurations:
 
 This is the list of Kubernetes versions that should have CAPA AMI support, based on the [CAPA AMI publication policy](../topics/images/built-amis.md#ami-publication-policy).
 
 It is generated by the command: `release-tool ami detect-k8s-release`
 
 <!-- $k8s_release_table -->
-
+| Minor Version | Patch Versions |
+| --- | --- |
+| `1.36` | `1.36.3`, `1.36.2`, `1.36.1`, `1.36.0` |
+| `1.35` | `1.35.7`, `1.35.6`, `1.35.5`, `1.35.4`, `1.35.3`, `1.35.2`, `1.35.1`, `1.35.0` |
+| `1.34` | `1.34.10`, `1.34.9`, `1.34.8`, `1.34.7`, `1.34.6`, `1.34.5`, `1.34.4`, `1.34.3`, `1.34.2`, `1.34.1`, `1.34.0` |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)